src/dashboard/ChartDashboardObject.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace luya\admin\dashboard;
+
+use Yii;
+use yii\helpers\Html;
+use yii\helpers\ArrayHelper;
+
+/**
+ * Fast generated Dashboard Objects.
+ *
+ * The default object is the default class for all {{luya\admin\base\Module::$dashboardObjects}} items without a class defintion.
+ *
+ * @author Oom <baqianxin@163.com>
+ * @blog  http://oom-cc.com
+ * @since 1.0.0
+ */
+class ChartDashboardObject extends BaseDashboardObject {
+
+
+    public $wrapperOptions = ['class' => 'card', 'tag' => 'div'];
+
+    /**
+     * @var string The Chart's option
+     *
+     */
+    public $options = '';
+
+    public function getOuterTemplate(){
```

Comment:
missing phpdoc and use psr2 style.

please take a look at how we indent and set the `{}` etc. => https://github.com/luyadev/luya/blob/master/docs/guide/luya-guideline.md#phpdoc